shaders/src/shading_reflections.fs

```diff
@@ -0,0 +1,23 @@
+/*
+ * screen-space reflection shading
+ */
+vec4 evaluateMaterial(const MaterialInputs material) {
+
+#if defined(MATERIAL_HAS_REFLECTIONS)
+    PixelParams pixel;
+    //getAnisotropyPixelParams(inputs, pixel);
+    vec4 color = vec4(0.0);
+    if (frameUniforms.ssrDistance > 0.0) {
```

Comment:
Do we still need to check `ssrDistance` here? Seems that it's redundant now that SSR is its own pass.

Reply:
yes because the material could have it enabled, while the View disables it. In this case we don't want to do the ray-marching.